Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```diff
@@ -83,6 +83,21 @@
       proper: true
       gender: female
 
+- type: entity
+  name: Puppy Ian
+  parent: MobCorgiPuppy
+  id: MobCorgiIanPup
+  description: Favourite puppy corgi. Awww.
+  components:
+  - type: Grammar
+    attributes:
+      proper: true
+      gender: male
+  - type: Butcherable # A puppy? You monster...
+    spawned:
+    - id: FoodMeatCorgi
```

Comment:
no corgi hide :trollface:

whats better than wearing ian suit? puppy ian suit

Reply:
You don't join the discord now this? You monster